packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -1,87 +1,62 @@
-import React, { useEffect, useContext } from 'react';
+import React, { useContext, useEffect } from 'react';
 import { Localized } from 'fluent-react';
 import { Plan } from '../../../store/types';
 import { AppContext } from '../../../lib/AppContext';
 
 import { metadataFromPlan } from '../../../store/utils';
+import { SelectorReturns } from '../../../store/selectors';
+
 import './index.scss';
+import PlanDetails from '../../../components/PlanDetails';
+import PaymentConfirmation from '../../../components/PaymentConfirmation';
+import Header from '../../../components/Header';
+
+const isMobile = window.matchMedia(
+  '(max-width: 520px), (orientation: landscape) and (max-width: 640px)'
```

Comment:
Will this also be updated with the new breakpoints?

We'll also need a debounced window resize event listener somewhere... A quick Google search [gave me this page](https://hackernoon.com/simplifying-responsive-layouts-with-react-hooks-19db73893a7a), I didn't read it in-depth but it seems like it could be a good direction?